plugins/kotlin/j2k/k1.new.post-processing/src/org/jetbrains/kotlin/idea/j2k/post/processing/allProcessings.kt

```diff
@@ -105,7 +105,6 @@ private val inspectionLikePostProcessingGroup = InspectionLikeProcessingGroup(
     inspectionBasedProcessing(ReplaceGetOrSetInspection(), checkInspectionIsEnabled = false),
     intentionBasedProcessing(ObjectLiteralToLambdaIntention(), writeActionNeeded = true),
     DestructureForLoopParameterProcessing(),
-    LiftReturnInspectionBasedProcessing(),
```

Comment:
Just in case, have you checked all the tests of LiftReturnOrAssignmentInspection (plugins/kotlin/idea/tests/testData/inspectionsLocal/liftOut)? I see that you have not added any new J2K tests, but I doubt that we have great existing test coverage here. Some tests should probably be ported.